admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -475,10 +475,136 @@ form#importDropzone {
 }
 
 /**
-* PrestaTrust modal
-**/
+ * PrestaTrust modal
+ **/
 
 #pstrust-img {
-    max-height: 64px;
-    max-width: 64px;
+  max-height: 64px;
+  max-width: 64px;
+}
+
+/**
+ *
```

Comment:
Miss comments but above all we're not suppose to add extra styling, we must use only bootstrap 4 and what is defined in UI Kit, we need to keep a trace on this somewhere so we coud extract it to UI Kit later, don't you think?